casaos-fix-docker-api-version/test-script.sh

```
@@ -0,0 +1,1633 @@
+  echo ""
+  
+  # Verify upgrade
+  local new_version=$(get_docker_version)
+  local new_api=$(get_docker_api_version)
+  
+  print_header "Upgrade Complete"
+  echo "New Docker version: $new_version"
+  echo "New API version:    $new_api"
+  echo ""
+  
+  if [[ "$new_api" == "1.45" ]] || [[ "$new_api" == "1.46" ]] || [[ "$new_api" > "1.44" ]]; then
```

Shellcheck (0.11.0)

[error] Decimals are not supported. Either use integers only, or use bc or awk to compare.

(SC2072)